types/ember__template/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for non-npm package @ember/template 3.0
+// Project: https://emberjs.com/api/ember/3.12/modules/@ember%2Ftemplate
+// Definitions by: Mike North <https://github.com/mike-north>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { SafeString } from '@ember/string/-private/handlebars';
```

Comment:
IMHO, we should move `@ember/string/-private/handlebars`  to *this* package (`@ember/template/-private/handlebars`).

Reply:
Ok, I'll go ahead and do that :+1: